src/pudl/metadata/classes.py

```diff
@@ -529,6 +536,22 @@ def from_id(cls, x: str) -> 'Encoder':
         """Construct an Encoder based on `Resource.name` of a coding table."""
         return cls(**cls.dict_from_id(x))
 
+    @classmethod
+    def from_code_id(cls, x: str) -> 'Encoder':
+        """Construct an Encoder based on looking up the name of a coding table directly in the codes metadata."""
+        return cls(**copy.deepcopy(CODE_METADATA[x]), name=x)
+
+    def to_rst(self, abs_csv_dir_path: DirectoryPath, rel_csv_dir_path: DirectoryPath, is_header: Bool) -> String:
+        """Ouput dataframe to a csv for use in jinja template. Then output to an RST file."""
+        abs_csv_filepath = Path(abs_csv_dir_path, f"{self.name}.csv")
```

Comment:
It's all a little tedious because the `csv-table` directive wants an absolute path that is relative to the source directory but pandas `to_csv` wants an absolute path. I tried to clean this up a little using `top_dir` and `csv_subdir`